app/template/[templateId]/q/[q_slug]/page.tsx

```diff
@@ -37,140 +40,157 @@ const QuestionEditPage: React.FC = () => {
   };
 
   return (
-    <div className="template-editor-container">
-      <div className="main-content">
-
-
-        {/* Using templateId from the URL to create a back link */}
-        <Link className="back-link-button"
-              href={`/template/${question.templateId}`}>
-          &larr; Back to template
-        </Link>
-
-
-        <h1>Edit: {question.text}</h1>
-
-
-        <Tabs>
-          <TabList aria-label="Question editing">
-            <Tab id="edit">Edit Question</Tab>
-            <Tab id="options">Options</Tab>
-            <Tab id="logic">Logic</Tab>
-          </TabList>
-          <TabPanel id="edit">
-            <Form>
-              <TextField
-                name="type"
-                type="text"
-                isRequired
-                value={question.type}
-              >
-                <Label>Type (required)</Label>
-                <Input disabled/>
-                <FieldError/>
-                <Button type="button">Change type</Button>
-              </TextField>
-
-              <TextField
-                name="question_text"
-                type="text"
-                isRequired
-                value={question.text}
-              >
-                <Label>Question text (required)</Label>
-                <Input
+    <>
```

Comment:
Similar to the Edit Section page, I couldn't tab to the `Options` and `Logic` tabs.

Reply:
See previous comments